web/src/redux/githubID/githubID.action.ts

```diff
@@ -0,0 +1,52 @@
+import GithubIDActionTypes from './githubID.types';
+import { Dispatch } from 'redux'
+import { ActionType } from 'typesafe-actions';
+
+export type GithubIDAction = ActionType<typeof requestGithubUsers>;
+
+export const requestGithubUsers = (persona: string) => ({
+  type: GithubIDActionTypes.GITHUB_USERS_REQUEST,
+  persona,
+});
+
+export const userExist = (persona: string) => ({
+  type: GithubIDActionTypes.GITHUB_USER_EXIST,
+  persona,
+});
+
+export const storeUser = (persona: string, payload: object) => ({
+  type: GithubIDActionTypes.GITHUB_USER_STORE_USER,
+  payload,
+  persona,
+});
+
+export const githubUserKeywordInput = (persona: Array<string>, payload: string) => ({
+  type: GithubIDActionTypes.GITHUB_USERS_INPUT,
+  payload,
+  persona,
```

Comment:
typically a redux action creator only has two top level fields `payload` and `type`. Everything else is within `payload`